scripts/windows/setup/IotEdgeSecurityDaemon.ps1

```diff
@@ -396,7 +400,10 @@ function Install-IoTEdge {
         [HashTable] $InvokeWebRequestParameters,
 
         # Restart if needed without prompting.
-        [Switch] $RestartIfNeeded
+        [Switch] $RestartIfNeeded,
+
+        # Bypass processor architecture check.
+        [Switch] $BypassArchCheck
```

Comment:
Consider using Skip instead of Bypass. Skip is used by DISM too for something similar. And Skip is in the approved list: https://docs.microsoft.com/en-us/powershell/developer/cmdlet/approved-verbs-for-windows-powershell-commands

Reply:
Good point. Fixed.